test/ui-testing/publishButton.test.js

```diff
@@ -6,9 +6,14 @@ beforeAll(async () => {
 });
 
 describe('Publish button', () => {
```

Comment:
OK, building from this - https://stackoverflow.com/questions/45948843/how-to-require-jquery-plugin-in-jest-test-file i'll try a simple fix. If it works, we may want to move that all into a shared config file so all Jest tests can access it. 

```suggestion
global.window = window
global.$ = require('jquery');

describe('Publish button', () => {
```